pkg/config/agent/config.go

```diff
@@ -246,3 +248,12 @@ type MulticastConfig struct {
 type EgressConfig struct {
 	ExceptCIDRs []string `yaml:"exceptCIDRs,omitempty"`
 }
+
+type MulticlusterConfig struct {
+	// Enable Multicluster which allow cross-cluster traffic between member clusters
+	// in a ClusterSet.
+	Enable bool `yaml:"enable,omitempty"`
+	// The Namespace where the Antrea Multi-cluster controller is running.
+	// 'kube-system' will be used by default if it's empty.
```

Comment:
The code actually reads the namespace from env, should shis say the namespace antrea-agent is running in?

Reply:
The default Namespace for Antrea MC controller is 'kube-system' the same as Agent, so I use the env to get Namespace if it's empty. But we can't guarantee user deploy MC controller to another Namespace, so here we provide a config for user to change it if it's not default one.

Reply:
Then do not say kube-system is the default, but say default to Antrea Agent Namespace.